Tests/SentryTests/Integrations/Performance/FramesTracking/SentryFramesTrackerTests.swift

```diff
@@ -588,13 +588,51 @@ class SentryFramesTrackerTests: XCTestCase {
     
     func testDealloc_CallsStop() {
         func sutIsDeallocatedAfterCallingMe() {
-            _ = SentryFramesTracker(displayLinkWrapper: fixture.displayLinkWrapper, dateProvider: fixture.dateProvider, dispatchQueueWrapper: SentryDispatchQueueWrapper(), keepDelayedFramesDuration: 0)
+            _ = SentryFramesTracker(displayLinkWrapper: fixture.displayLinkWrapper, dateProvider: fixture.dateProvider, dispatchQueueWrapper: SentryDispatchQueueWrapper(), notificationCenter: TestNSNotificationCenterWrapper(), keepDelayedFramesDuration: 0)
         }
         sutIsDeallocatedAfterCallingMe()
         
         XCTAssertEqual(1, fixture.displayLinkWrapper.invalidateInvocations.count)
     }
     
+    func testFrameTrackerPauses_WhenAppGoesToBackground() {
+        let sut = fixture.sut
+        sut.start()
+        
+        var callbackCalls = 0
+        let listener = FrameTrackerListener()
+        listener.callback = {
+            callbackCalls += 1
+        }
+        sut.add(listener)
```

Comment:
I don't think this is actually necessary here? We don't assert the number of `callbackCalls`. The one below does, maybe we just add the assertion here too.